wallet/how-to/connect/index.md

```diff
@@ -1,166 +1,313 @@
 ---
 description: Connect to MetaMask via EIP-6963.
 sidebar_position: 1
+toc_max_heading_level: 4
 ---
 
 # Connect to MetaMask
 
 You can connect your dapp to users' MetaMask wallets by detecting MetaMask in their browsers and
 connecting to their accounts.
 
-This page describes how to connect to MetaMask using the wallet detection mechanism introduced by
-[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963).
-EIP-6963 enables [wallet interoperability](../../concepts/wallet-interoperability.md), and shifts
-dapps from relying solely on [`window.ethereum`](detect-metamask.md) for wallet detection.
-If a user has multiple wallet browser extensions installed, you can detect multiple wallets and
-connect to each one without conflict.
+The recommended method for connecting to MetaMask or other browser wallets is using the wallet
```

Comment:
I think this is in passive voice. Maybe this can be rewritten for active voice, such as:

We recommend using the wallet detection mechanism introduced by [EIP-6963](https://eips.ethereum.org/EIPS/eip-6963) for connecting to MetaMask and other browser wallets. This approach allows you to detect and connect to multiple wallets and without conflicts.